src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="body">
+    <section
+      class="card-info"
+      data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="card-info__image">
+      <p class="card-info__name">
```

Comment:
```suggestion
    <section
      class="card-info"
      data-qa="card"
    >
      <img
        src="./images/imac.jpeg"
        alt="imac"
        class="card-info__image"
      >
      <p class="card-info__name">
```
fix code style
<img width="990" alt="Screenshot 2022-08-26 at 09 20 47" src="https://user-images.githubusercontent.com/94289038/186835815-a6402c89-d435-4ea7-ae40-bc7466443bee.png">